src/cpp/rtps/transport/shared_mem/SharedMemGlobal.hpp

```diff
@@ -99,7 +99,7 @@ class SharedMemGlobal
 
     struct PortNode
     {
-        alignas(8) std::atomic<std::chrono::high_resolution_clock::rep> last_listeners_status_check_time_ms;
+        alignas(8) std::atomic<std::chrono::steady_clock::rep> last_listeners_status_check_time_ms;
```

Comment:
This is changing the behaviour on a structure that is used in the shared memory transport.

This would mean that current versions of the code would misinterpret the meaning of this field.

I suggest putting the changes on this file on a separate PR.

Reply:
I went ahead and reverted the changes in commit f94d111b6b2af4af3427a233edc0e815fe888f39